### PR TITLE
Updated Article

### PR DESCRIPTION
**Pull request by mlafleur**
The documentation was unintentionally implying that the `Cache-Control` header only applied to Add-ins published to AppSource. To clarify that this applies to any add-in, I've broken this (and `HTTPS`) into their own section. 

- Added Hosting Requirements section (separate from AppSource) covering `Cache-Control` and `HTTPS`
- Cleaned up table formatting
- Updated blockquote Notes to use DFM [!NOTE]
- Placed manifest examples in tabs to reduce length/noise

### CODE REVIEW

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802062104141114-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Q51Ufq8Y46s1Glk6wTNfsikb%2Bh%2FMzpFv8qTzbG%2BtPUA%3D&st=2018-02-06T21%3A00%3A07Z&se=2018-03-09T21%3A05%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/requirements-for-running-office-add-ins.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ To be able to install and run any Office Add-in, you first need to deploy the ma
 
 For all types of add-ins (content, Outlook, and task pane add-ins and add-in commands), you need to deploy your add-in's webpage files to a web server, or web hosting service, such as [Microsoft Azure](../publish/host-an-office-add-in-on-microsoft-azure.md).
 
+Add-ins and assets must be served over secure transport (`HTTPS`) in both Development and Production environments. Self-signed certificates can be used for development and testing, so long as the certificated is trusted on the local machine.
```
Comment (Reezaali) [inline docs/concepts/requirements-for-running-office-add-ins.md]: Typo - certificated should be certificate

File: docs/develop/add-in-manifests.md [hunk 2]
```diff
@@ -6,62 +6,97 @@ ms.date: 01/23/2018
...
+[set*]: http://msdn.microsoft.com/en-us/library/1506daa1-332c-30e1-6402-3371bcd0b895%28Office.15%29.aspx
+[sets (mailapprequirements)*]: http://msdn.microsoft.com/en-us/library/2a6a2484-eeee-37e4-43bc-c185e8ae0d1d%28Office.15%29.aspx
+[form*]: http://msdn.microsoft.com/en-us/library/77a8ac83-c22b-1225-4fc4-ba4038b68648%28Office.15%29.aspx
+[formsettings*]: http://msdn.microsoft.com/en-us/library/0d1a311d-939d-78c1-e968-89ddf7ebc4b4%28Office.15%29.aspx
+[sets (requirements)*]: http://msdn.microsoft.com/en-us/library/509be287-b532-87c6-71ac-64f3a4bbd3af%28Office.15%29.aspx
+[hosts*]: http://msdn.microsoft.com/library/f9a739c1-3daf-c03a-2bd9-4a2a6b870101%28Office.15%29.aspx
+
+## Hosting Requirements
+
+All image URIs, such as those used for [Add-in Commands][], **must support caching**. The server hosting the image should not return a `Cache-Control` header specifying `no-cache`, `no-store`, or similar options in the HTTP response.
```
Comment (Reezaali) [inline docs/develop/add-in-manifests.md]: Can you remove the bold style around "must support caching" please? Thanks.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802062156380619-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2B7nOz2UN1ApTDBY%2BHYBiVXjzWJZA1y0ht0mcd4lJtDM%3D&st=2018-02-06T21%3A52%3A31Z&se=2018-03-09T21%3A57%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802091854170421-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gcBjXGIVQXe7y3mspCpOoePpRrn5Vc4G6ZE2JT8v1nU%3D&st=2018-02-09T18%3A50%3A12Z&se=2018-03-12T18%3A55%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802092008470950-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=K51LRViWZIcx0sTr7VQQCO5iwji%2F8hONyFZtffgqoO8%3D&st=2018-02-09T20%3A04%3A42Z&se=2018-03-12T20%3A09%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802092010344901-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bwqhNMIL%2B%2Bb9om8hR2wsErkKHB6upcwa7cj5bCQ6bjw%3D&st=2018-02-09T20%3A06%3A29Z&se=2018-03-12T20%3A11%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802092022557969-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Gh5%2F3vveVJcu3nyGmeV8zEMnT%2BsGs6thvD737GhPVyk%3D&st=2018-02-09T20%3A18%3A50Z&se=2018-03-12T20%3A23%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802092031236081-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4M8hw%2Bz878XhVlEQkAUQO%2BSFaKuiSJikgfHpzTUE0uo%3D&st=2018-02-09T20%3A27%3A17Z&se=2018-03-12T20%3A32%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/includes/https-guidance.md](https://github.com/mlafleur/office-js-docs-pr/blob/marcla-hostingreq/docs/includes/https-guidance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201802092038032009-85%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=a8SHV03Cr5JfwNBymbbvkKiXUI7qUT633sbg8WrHJPk%3D&st=2018-02-09T20%3A33%3A57Z&se=2018-03-12T20%3A38%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**